src/OrbitQt/CaptureOptionsDialog.ui

```diff
@@ -282,6 +282,60 @@ p, li { white-space: pre-wrap; }
+               <property name="toolTip">
+                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;With Wine 7 (and Proton 7), the implementation of Windows system calls operate on a different stack. Orbit's DWARF stack unwinding requires a copy of the stack to successfully unwind the callstack offline. This copy will not contain the &amp;quot;Windows user-space&amp;quot; stack, and thus unwinding will fail on the bridge method (__wine_syscall_dispatcher).&lt;/p&gt;&lt;p&gt;With this option, Orbit will not perform any special handling for Wine system calls.&lt;/p&gt;&lt;p&gt;On Wine 7, Orbit will likely fail unwinding after the &amp;quot;__wine_syscall_dispatcher&amp;quot; function and report an unwinding error.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+               </property>
+               <property name="text">
+                <string>No special handling ⓘ</string>
+               </property>
+              </widget>
+             </item>
+             <item>
+              <widget class="QRadioButton" name="wineStopRadioButton">
+               <property name="toolTip">
+                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;With Wine 7 (and Proton 7), the implementation of Windows syste´m calls operate on a different stack. Orbit's DWARF stack unwinding requires a copy of the stack to successfully unwind the callstack offline. This copy will not contain the &amp;quot;Windows user-space&amp;quot; stack, and thus unwinding will fail on the bridge method (__wine_syscall_dispatcher).&lt;/p&gt;&lt;p&gt;With this option, Orbit will stop unwinding when reaching the &amp;quot;__wine_syscall_dispatcher&amp;quot; function and will not report an unwinding error.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+               </property>
+               <property name="text">
+                <string>Only unwind the &quot;kernel&quot; callstack (fast) ⓘ</string>
+               </property>
+              </widget>
+             </item>
+             <item>
+              <widget class="QRadioButton" name="wineRecordRadioButton">
+               <property name="toolTip">
+                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;With Wine 7 (and Proton 7), the implementation of Windows system calls operate on a different stack. Orbit's DWARF stack unwinding requires a copy of the stack to successfully unwind the callstack offline. This copy will not contain the &amp;quot;Windows user-space&amp;quot; stack, and thus unwinding will fail on the bridge method (__wine_syscall_dispatcher).&lt;/p&gt;&lt;p&gt;With this option, Orbit will record the &amp;quot;Windows user-space&amp;quot; on every call to &amp;quot;__wine_syscall_dispatcher&amp;quot; to have it available during unwinding.&lt;/p&gt;&lt;p&gt;While this approach yields complete callstacks, the approach introduces significant performance overhead.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
In the last sentence, there is a repetition of the word "approach": "While this approach yields complete callstacks, _it_ introduces significant performance overhead."

Reply:
Done.